google_api_availability/lib/google_api_availability.dart

```diff
@@ -1,4 +1,4 @@
 library google_api_availability;
 
 export 'src/google_api_availability.dart';
-export 'src/models/google_play_services_availability.dart';
+export 'package:google_api_availability_platform_interface/google_api_availability_platform_interface.dart';
```

Comment:
I don't think this is correct, why would you export a different package?

Reply:
We export a model that can be returned from a function call. As the model resides in the platform code, we need to export it here (as the end-user might not have a dependency on the platform interface).

I changed the export to export only the model itself, as I do agree this is exposes too much.